tools/release-ci/cmd/sidecar.go

```diff
@@ -0,0 +1,89 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strconv"
+	"time"
+
+	"cloud.google.com/go/storage"
+	"github.com/spf13/cobra"
```

Comment:
Do we care about the additional size added by cobra on this? Do we care about the size of the sidecar at all?